pipeline/07-export.R

```diff
@@ -271,8 +271,40 @@ assessment_card <- dbGetQuery(
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 message("Preparing data for Desk Review export")
 
+# Get the number of apartments for class 211 and 212 properties. Since
+# char_apts is recorded on the card level, we need to aggregate them to
+# the PIN level
+num_apts_by_pin <- assessment_card %>%
+  filter(char_class == "211" | char_class == "212") %>%
+  select(meta_pin, char_apts) %>%
+  # Convert the long format for char_apts to a shorter one that's easier
+  # to scan in a spreadsheet
+  mutate(
+    char_apts = case_when(
+      char_apts == "None" | is.na(char_apts) ~ "Missing",
+      char_apts == "Two" ~ "2",
+      char_apts == "Three" ~ "3",
+      char_apts == "Four" ~ "4",
+      char_apts == "Five" ~ "5",
+      char_apts == "Six" ~ "6",
+      TRUE ~ "Missing"
+    )
+  ) %>%
```

Comment:
Is there an equivalent function to do this in the `ccao` package? I thought about using `vars_recode`, but the logic is just unusual enough (particularly the `Missing` stuff) that it didn't quite fit, so I just went with the verbose version instead.

Reply:
There isn't at present. I think this is totally fine. I have an extremely weird follow up on this for next week...